git_acp/git/classification.py

```
@@ -567,8 +559,11 @@ def _collect_signals(
     numstat: dict[str, tuple[int, int]] = {}
     try:
         numstat = get_numstat(config)
-    except (GitError, Exception):
-        pass
+    except GitError:
+        pass  # expected: no staged/unstaged changes
+    except Exception as err:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)